ci_config.json

```diff
@@ -851,5 +851,7 @@
       "wayland:tests=false",
       "wayland:dtd_validation=false"
     ]
+  },
+  "tree_sitter": {
```

Comment:
This is not needed if there contents is empty.

Reply:
fixed